generator/download-repos.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# get googleapis repo
+git clone https://github.com/googleapis/googleapis.git
+
+# Prepare `gapic-generator-java` with Spring generation ability.
+# If keeping a copy in this repo, this is not needed.
+# Checkout `gapic-generator-java`
+git clone https://github.com/googleapis/gapic-generator-java.git
+# get into gapic and checkout branch to use
+cd gapic-generator-java
+git checkout c51d51bc495ea1284087b66c369dedf741dd4824
+# go back to previous folder
+cd -
+
+
+cd googleapis
+# fix googleapis committish for test/dev purpose
+git checkout f88ca86
+# todo: change to local repo --> gapic
+# very not stable change - todo: change to search and replace.
```

Comment:
Dropping in a quick note here for the update merged in #1315, since it might not get sync'd or show up as conflict after the file split.